resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -206,6 +206,7 @@ serverless.application.deploy.error.invalid_runtime=Invalid runtime {0} defined
 serverless.application.deploy.error.invalid_runtime_group=Cannot find runtime group for runtime {0} in template {1}
 serverless.application.deploy.error.unsupported_runtime_group=The runtime {0} is not supported for deploying the SAM template {1}
 serverless.application.deploy.error.empty_runtime=The runtime is not defined for all functions in template {0}
+sam.executable.not_found=File not found: "{0}"
```

Comment:
I guess we should rename it to `file.not_found` instead of `sam.executable.not_found`